### PR TITLE
Update highcharts

### PR DESCRIPTION
**Pull request by Norreman**

### CODE REVIEW

**Review by marcelkilian — CHANGES_REQUESTED** [review]
I commented some lines and more importanly the build fails. pls fix

File: dist/ChartComponent.d.ts [hunk 1]
```diff
@@ -22,7 +22,6 @@ export declare class ChartComponent {
     chart: any;
     element: ElementRef;
     highchartsService: HighchartsService;
-    test: any;
```
Comment (marcelkilian) [inline dist/ChartComponent.d.ts:25]: Why did you remove it and what did it do in the first place?

File: package.json [hunk 2]
```diff
@@ -50,7 +50,7 @@
     "zone.js": "^0.7.6"
   },
   "dependencies": {
-    "@types/highcharts": "^4.2.47",
-    "highcharts": "^5.0.7"
+    "@types/highcharts": "^5.0.12",
```
Comment (marcelkilian) [inline package.json:53]: Those are type descriptions for version 5.0.12. Sadly there is no update yet.

File: dist/ChartComponent.d.ts (hunk 1 above)
Comment (Norreman) [reply dist/ChartComponent.d.ts:25]: Its in the dist folder and this is generated. I haven't delete it on my own.

File: package.json (hunk 2 above)
Comment (Norreman) [reply package.json:53]: Yes, sadly.

**Review by marcelkilian — APPROVED** [review]

**Pull request merged**